src/lib.rs

```diff
@@ -59,6 +60,33 @@ pub enum DisplayState {
     Editable,
 }
 
+/// Configure spacing and separators for the view
+#[derive(Debug, Clone)]
+pub struct HexViewConfig {
+    /// Number of bytes per line
+    pub bytes_per_line: usize,
+    /// Number of bytes per group
+    pub bytes_per_group: usize,
+    /// Separation between groups
```

Comment:
If we go for the `byte_group_separator` field, then this one will be obsolete, what do you think?

Reply:
mmmm, not sure, I was thinking that the proposal for group separator would be same as we now have the separator between hex and ascii, which is a fixed value of a fixed length. In this case, if we do this, then if someone wants four spaces, they would need to set the group separator as "    ". So it is doable I guess, both ways would work. I like it how it  is right now better. And actually, even if adding the group separator, if we don't remove this one, then it's more flexible, even if it adds two ways of accomplishing the same thing:

1- group separator: " ", separation: 1 --> ab cd ef
2- group separator: " ", separation: 4 --> ab    cd    ef
3- group separator: "   ", separation: 1 --> ab    cd    ef
4- group separator: "<>", separation: 2 -> ab<><>cd<><>ef
5- group separator: "<><>", separation: 1 -> ab<><>cd<><>ef

If we leave it as it is, we can do 1 and 2
If we replace count for separator we can do 1, 3 and 5
If we keep both we can do all those 5.

2 and 3 are the same; 4 and 5 are the same, so both replacing or adding would allow all options. keeping the count just maintains that flexibility for the interface, which in my opinion is clearer since the " " is probably going to be the separator used 99.999999999999% of the times (the number of 9s is accurate :) ).

Reply:
Honstely, I would prefer having a new separator field 🤐 
Of course, most of the people would have a ` ` as separator, that's why it should be the `Default::default` case. Would you like to change the proposal to add a new field `byte_group_separator` and make it ` ` as default? The doc for it should be something like this:

```
Controls the separator between the hex groups in the data output.

Default is ` ` (0x20)
```